Authorization/Authorization/Presentation/Registration/SignUpViewModel.swift

```diff
@@ -193,6 +193,7 @@ public class SignUpViewModel: ObservableObject {
             analytics.userLogin(method: authMethod)
             isShowProgress = false
             router.showMainOrWhatsNewScreen(sourceScreen: sourceScreen)
+            NotificationCenter.default.post(name: .userAuthorized, object: nil)
```

Comment:
Right. Fixed ✅

Reply:
I don't understand why it is redundant.
It looks reasonable for me to be placed here.

Reply:
The following code is same in both the success methods `loginOrRegister` and `registerUser`. Anyhow we can live with it because it's not changed under the scope of this PR. 
```
let user = try await interactor.login(externalToken: response.token, backend: backend)
            analytics.identify(id: "\(user.id)", username: user.username, email: user.email)
            analytics.userLogin(method: authMethod)
            isShowProgress = false
            router.showMainOrWhatsNewScreen(sourceScreen: sourceScreen)
            NotificationCenter.default.post(name: .userAuthorized, object: nil)
```

Reply:
Sure, we can address this later in some refactoring